cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,18 @@
-/// <reference types="cypress" />
+/// <reference types="Cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => {
+    cy.visit('/#/login');
+  });
+
   it('should provide an ability to log in', () => {
+    cy.get('[placeholder = "Email"]').type('test_user_1@email.com');
 
+    cy.get('[placeholder = "Password"]').type('123456789');
+    cy.contains('[type="submit"]', 'Sign in')
+      .click();
+    cy.contains('My Settings')
```

Comment:
There is no such element as 'My Settings' on the page after user logs in.
Instead you're supposed to assert if your username appears in the top right corner:
![image](https://github.com/mate-academy/qa_cypress_e2e_intro/assets/37079719/f9d51999-0001-43b9-b849-ccfc2b340cde)